comms/rabbitmq.go

```diff
@@ -683,19 +683,19 @@ func (rmq rabbitMQComms) ListenForGroupTasks(groupName string, dereg chan bool)
 		for d := range msgs {
 
 			// Unmarshal into BaseTaskMessage to determine type
-			var base_msg tasks.BaseTask
-			err = json.Unmarshal(d.Body, &base_msg)
+			var baseMsg tasks.BaseTask
+			err = json.Unmarshal(d.Body, &baseMsg)
 			// TODO(mierdin): Need to handle this error
 
 			log.Debugf("Agent task received: %s", d.Body)
 
 			// call agent task method based on type
-			switch base_msg.Type {
+			switch baseMsg.Type { // TODO: remove switch?
```

Comment:
Actually a future design for this may need it again. Can you remove this TODO and then follow my next comment?